### PR TITLE
Chore/embl logo update

### PR DESCRIPTION
**Pull request by khawkins98**
This uses the latest logos that tabe sent around and turned into a bit more than first anticipated:

- Brings in the suggestions from (BUG - vf-global-header site link, misc related issues #490) and your branch
- Drops the PNG logos (no need for them)

### CODE REVIEW

File: components/embl-logo/assets/logo-medium.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...79 0 00-3.78-3.78zm0 11.16A3.78 3.78 0 10131 54a3.79 3.79 0 00-3.78-3.75zm.01 11.19a3.79 3.79 0 103.77 3.79 3.79 3.79 0 00-3.77-3.79zM147 5.6a3.79 3.79 0 103.78 3.79A3.79 3.79 0 00147 5.6zm0 11.17a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm19.85-11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zM147 50.25a3.78 3.78 0 103.81 3.75 3.79 3.79 0 00-3.81-3.75zm0 11.19a3.79 3.79 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm-39.6-44.67a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78zm0 11.16a3.78 3.78 0 103.78 3.75 3.78 3.78 0 00-3.78-3.75zm9.91-39.1a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78zm0 11.15a3.79 3.79 0 103.78 3.79 3.79 3.79 0 00-3.78-3.79zm0 11.16a3.78 3.78 0 103.78 3.78 3.78 3.78 0 00-3.78-3.78z"/><path d="M117.31 44.61a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78z" fill="#e40046"/><path class="cls-2" d="M117.31 55.76a3.78 3.78 0 103.78 3.78 3.79 3.79 0 00-3.78-3.78z"/></svg>
```
Comment (sturobson) [inline components/embl-logo/assets/logo-medium.svg]: the state of this SVG file 😭 - I don't think we'd need the classnames or the ids? I tried running this through [SVGOMG](https://jakearchibald.github.io/svgomg/) but it didn't show as an SVG :(

File: components/embl-logo/embl-logo.scss [hunk 2]
```diff
@@ -1,9 +1,13 @@
 // embl-logo
 
 .embl-logo {
+  background-image: url('../assets/embl-logo/assets/logo.png');
```
Comment (sturobson) [inline components/embl-logo/embl-logo.scss]: `.png` here?

File: components/vf-logo/vf-logo.scss [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
-    background-image: url('../assets/vf-logo/assets/logo-medium.svg');
     height: 28px;
   }
-}
-@media (min-width: 999px) {
-  .vf-logo--responsive {
-    background-image: url('../assets/vf-logo/assets/logo.svg');
+
+  @media (min-width: 999px) {
+    --logo-spacing: 16px;
```
Comment (sturobson) [inline components/vf-logo/vf-logo.scss]: should we namespace this to `--vf-logo-spacing` and also should we be explicit? `--vf-logo--margin-right`?

File: components/embl-logo/embl-logo.scss (hunk 2 above)
Comment (khawkins98) [reply components/embl-logo/embl-logo.scss]: Forgot to remove some of the PNG stuff, should be sorted now.

File: components/vf-logo/vf-logo.scss (hunk 3 above)
Comment (khawkins98) [reply components/vf-logo/vf-logo.scss]: Sounds like that fits our methodology, I'll update.

Comment (sturobson) [reply components/vf-logo/vf-logo.scss]: As it's a CSS custom property set within the components classname it's localised enough that this shouldn't be a problem.

File: components/embl-logo/assets/logo-medium.svg (hunk 1 above)
Comment (sturobson) [reply components/embl-logo/assets/logo-medium.svg]: let's get this in, and I can look at fixing the Adobe-ified SVG stuff later

Comment (khawkins98) [reply components/embl-logo/assets/logo-medium.svg]: I dare not tread within ... at least until I know no further revisions to the logo. 😭

**Pull request merged**